For compat-wireless based  builds use compat_base_tree_version in  version.h
When building with compat-wireless mechanism, the version info captured in version.h reflects target kernel and not the wl12xx version.
Signed-off-by: Vishal Mahaveer <vishalm@ti.com>

diff  a/drivers/net/wireless/wl12xx/Makefile b/drivers/net/wireless/wl12xx/Makefile

--- a/drivers/net/wireless/wl12xx/Makefile 2012-08-12 18:18:10.000000000 +0300
+++ b/drivers/net/wireless/wl12xx/Makefile 2012-08-13 13:55:11.000000000 +0300
@@ -3,12 +3,16 @@
 
 define filechk_version.h
 	(echo 'static const char *wl12xx_timestamp = __TIMESTAMP__;'; \
-	echo 'static const char *wl12xx_git_head = \
-			"$(shell git describe --dirty)";')
+         echo '#ifdef COMPAT_BASE_TREE_VERSION'; \
+         echo 'static const char *wl12xx_git_head =  COMPAT_BASE_TREE_VERSION;'; \
+         echo '#else'; \
+         echo 'static const char *wl12xx_git_head = \
+                        "$(shell git describe --dirty)";'; \
+         echo '#endif')
 endef
 
-$(obj)/version.h: .git/HEAD .git/index .git/refs/tags
-	@$(call filechk,version.h)
+#$(obj)/version.h: .git/HEAD .git/index .git/refs/tags
+#	@$(call filechk,version.h)
 
 $(obj)/main.c: $(src)/version.h
 
